R/FilteredDataset.R

```diff
@@ -417,11 +442,20 @@ FilteredDataset <- R6::R6Class( # nolint
           shiny::observeEvent(self$get_filter_state(), {
             if (length(self$get_filter_state()) == 0) {
               shinyjs::hide("remove_filters")
+              shinyjs::hide("collapse")
+              shinyjs::hide("filter_count_ui")
+              shinyjs::show("filters")
```

Comment:
If we move these two out of the `if` (but keep them in the `observeEvent` then this would mean the filters always stop being collapsed when a filter is added or removed - which may be what we want.

Reply:
+1 I would prefer the filters to be re-collapsed then, the other way it feels like nothing has happened when you add the filter.

Reply:
regarding "(but not when Active Filter Variables is collapsed)." - we have https://github.com/insightsengineering/teal.slice/issues/6 it might be something you want to take on?

Reply:
changed